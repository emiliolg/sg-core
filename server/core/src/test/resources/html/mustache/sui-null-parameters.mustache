<!DOCTYPE html>
<html sg-view="tekgenesis.service.html.Views" sg-params="text:String, number:Int, list:String*">
<head>
    <title>Parameters examples</title>
</head>
<body>

<h2>No value, no fail</h2>
<div>No Text: {{text}}</div>
<div>No List: {{list}}</div>
<div>Zero Number: {{number}}</div>

<h2>No value, no true condition</h2>
<div>Empty Text Condition: {{#text}}Text?{{/text}}</div>
<div>Zero Number Condition: {{#number}}Not Zero?{{/number}}</div>
<div>Empty List Condition: {{#list}}List with values?{{/list}}</div>

</body>
</html>
