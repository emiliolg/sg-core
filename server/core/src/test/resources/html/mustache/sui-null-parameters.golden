<!DOCTYPE html>
<html sg-view="tekgenesis.service.html.Views" sg-params="text:String, number:Int, list:String*">
<head>
    <title>Parameters examples</title>
</head>
<body>

<h2>No value, no fail</h2>
<div>No Text: </div>
<div>No List: </div>
<div>Zero Number: 0</div>

<h2>No value, no true condition</h2>
<div>Empty Text Condition: </div>
<div>Zero Number Condition: </div>
<div>Empty List Condition: </div>

</body>
</html>
